# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/parent/FileSystemManagerParent.h
# Commit: a2f6102fc8ea
# Full Hash: a2f6102fc8ea11d4d9b3f2183ef6b8641e0ceeb5
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 3
# Description:
#   Bug 1786465 - Start using FileSystemDataManager in FileSystemManagerParent; r=dom-storage-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D155552
# ==============================================================================

diff -r 37b7616c8b42 -r a2f6102fc8ea dom/fs/parent/FileSystemManagerParent.h
--- a/dom/fs/parent/FileSystemManagerParent.h	Wed Aug 31 09:34:54 2022 +0000
+++ b/dom/fs/parent/FileSystemManagerParent.h	Wed Aug 31 09:34:55 2022 +0000
@@ -29,7 +29,9 @@
 
 class FileSystemManagerParent : public PFileSystemManagerParent {
  public:
-  FileSystemManagerParent(TaskQueue* aTaskQueue, const EntryId& aRootEntry);
+  FileSystemManagerParent(TaskQueue* aTaskQueue,
+                          RefPtr<fs::data::FileSystemDataManager> aDataManager,
+                          const EntryId& aRootEntry);
 
   mozilla::ipc::IPCResult RecvGetRootHandleMsg(
       GetRootHandleMsgResolver&& aResolver);